test-cloud/test-cloud-remote/src/test/java/org/kie/cloud/integrationtests/s2i/KieServerS2iDroolsIntegrationTest.java

```diff
@@ -0,0 +1,135 @@
+                .withContainerDeployment(KIE_CONTAINER_DEPLOYMENT)
+                .withSourceLocation(gitProvider.getRepositoryUrl(repositoryName), REPO_BRANCH, DEPLOYED_KJAR.getName())
+                .withDroolsServerFilterClasses(false)
+                .build();
+
+        return deploymentScenarioFactory.getGenericScenarioBuilder()
+                .withKieServer(kieServerS2Isettings)
+                .build();
+    }
+
+    @Before
+    public void setUp() throws ClassNotFoundException {
+        deploymentScenario.getKieServerDeployments().get(0).setRouterTimeout(Duration.ofMinutes(5));
+
+        MavenDeployer.buildAndInstallMavenProject(ClassLoader.class.getResource("/kjars-sources/stateless-session").getFile());
+        kjarClassLoader = KieServices.Factory.get().newKieContainer(RELEASE_ID).getClassLoader();
+
+        extraClasses = Collections.singleton(Class.forName(PERSON_CLASS_NAME, true, kjarClassLoader));
+        kieServicesClient = KieServerClientProvider.getKieServerClient(deploymentScenario.getKieServerDeployments().get(0), extraClasses);
+        ruleClient = kieServicesClient.getServicesClient(RuleServicesClient.class);
+    }
+
+    @After
+    public void deleteRepo() {
+        gitProvider.deleteGitRepository(repositoryName);
+    }
+
+    @Test
+    public void testContainerAfterExecServerS2IStart() {
+        List<KieContainerResource> containers = kieServicesClient.listContainers().getResult().getContainers();
+        assertThat(containers).hasSize(1);
+
+        KieContainerResource container = containers.get(0);
+        assertThat(container).isNotNull();
+        assertThat(container.getContainerId()).isEqualTo(CONTAINER_ID);
+
+        ReleaseId containerReleaseId = container.getResolvedReleaseId();
+        assertThat(containerReleaseId).isNotNull();
+        assertThat(containerReleaseId.getGroupId()).isEqualTo(DEPLOYED_KJAR.getGroupId());
+        assertThat(containerReleaseId.getArtifactId()).isEqualTo(DEPLOYED_KJAR.getName());
+        assertThat(containerReleaseId.getVersion()).isEqualTo(DEPLOYED_KJAR.getVersion());
+
+        List<Command<?>> commands = new ArrayList<Command<?>>();
+        BatchExecutionCommand executionCommand = commandsFactory.newBatchExecution(commands, KIE_SESSION);
+
+        Object person = KieServerReflections.createInstance(PERSON_CLASS_NAME, extraClasses.iterator().next().getClassLoader(), PERSON_NAME, "");
+        commands.add(commandsFactory.newInsert(person, PERSON_OUT_IDENTIFIER));
+
+        ServiceResponse<ExecutionResults> reply = ruleClient.executeCommandsWithResults(CONTAINER_ID, executionCommand);
+        assertEquals(ServiceResponse.ResponseType.SUCCESS, reply.getType());
+
+        ExecutionResults actualData = reply.getResult();
+        Object value = actualData.getValue(PERSON_OUT_IDENTIFIER);
+
+        assertEquals("Expected surname to be set to 'Vader'",
```

Comment:
Can you replace this junit Assert with assertj?

Reply:
Done.